src/routers/product.router.ts

```diff
@@ -29,7 +29,7 @@ productRouter.get("/:id", async (req: Request, res: Response) => {
 });
 
 productRouter.post("", async (req: Request, res: Response) => {
-  const userId: number = 1;
+  const userId: number = 5;
```

Comment:
I don't get why this line ?

Reply:
Its a temporary solution until the architecture of the front end is done. thnaks